paasta_tools/cli/schemas/kubernetes_schema.json

```
@@ -190,7 +190,124 @@
                     "type": "string"
                 },
                 "autoscaling": {
-                    "type": "object"
+                    "type": "object",
+                    "properties": {
+                        "metrics_provider": {
+                            "enum": [
+                                "uwsgi",
+                                "mesos_cpu",
+                                "cpu",
+                                "piscina",
+                                "http",
+                                "arbitrary_prometheus"
```

looks like you might have renamed this during development? in the python code we use `arbitrary_promql`
```suggestion
                                "arbitrary_promql"
```